feature-libs/product-configurator/rulebased/cpq/rest/cpq-configurator-rest.module.ts

```diff
@@ -30,26 +20,6 @@ import { defaultCpqConfiguratorEndpointConfig } from './default-cpq-configurator
       useClass: CpqConfiguratorRestAdapter,
       multi: true,
     },
-    {
```

Comment:
This is breaking, we must not do this. We can relocate a module to a different folder, but we can't remove stuff from a module. Imagine a customer importing this one and expecting to have the normalizers registered

Reply:
ok - i restored it, so this is provided in both modules - the old rest and the new common module.
or should i removed it from the new common module?

Reply:
Better remove it from the new common module. And add a comment referring to https://jira.tools.sap/browse/CXSPA-3392 so that we not forget to migrate that in the next major.
You might add a comment in the JIRA issue stating that we do this change and provide the respective upgrade documentation